src/app/features/first-login-wizard/first-login-wizard.component.html

```diff
@@ -0,0 +1,62 @@
+<div class="govuk-grid-row govuk-!-margin-1 govuk-!-margin-top-4">
+  <div class="asc-summary-box">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full govuk-util__align-right">
+        <a class="govuk-button govuk-button--link" href="/dashboard">Close</a>
+      </div>
+    </div>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half">
+        <h1 class="govuk-!-font-weight-regular govuk-!-margin-top-0">Help to get you started</h1>
+        <h2 class="govuk-heading-m govuk-!-margin-bottom-2">
+          {{ wizards[currentIndex].title }}
+        </h2>
+        <div class="asc-cms-content" [innerHTML]="wizards[currentIndex].content"></div>
+      </div>
+      <div class="govuk-grid-column-one-half">
+        <img
+          class="asc-shadow-box"
+          src="{{ imageUrl }}{{ wizards[currentIndex].image }}"
+          alt="{{ wizards[currentIndex].title }}"
+        />
+      </div>
+    </div>
+    <div class="govuk-grid-row">
+      <div class="govuk-util__inline-block govuk-grid-column-one-half govuk-util__align-left">
+        <a
+          *ngIf="!isFirst"
+          class="asc-chevron-before asc-no-underline govuk-!-font-size-27 govuk-button--link govuk-!-margin-top-3"
+          href="#"
+          (click)="previousWizard($event)"
+          >Previous</a
+        >
+        <br />
+        <a
+          *ngIf="!isFirst"
+          href="#"
+          (click)="previousWizard($event)"
+          class="govuk-button--link govuk-!-margin-left-2"
+          >{{ wizards[currentIndex - 1].title }}</a
+        >
+      </div>
```

Comment:
Why not put the `*ngIf` round the whole div?
```suggestion
      <div class="govuk-util__inline-block govuk-grid-column-one-half govuk-util__align-left" *ngIf="!isFirst">
        <a
          class="asc-chevron-before asc-no-underline govuk-!-font-size-27 govuk-button--link govuk-!-margin-top-3"
          href="#"
          (click)="previousWizard($event)"
          >Previous</a
        >
        <br />
        <a
          href="#"
          (click)="previousWizard($event)"
          class="govuk-button--link govuk-!-margin-left-2"
          >{{ wizards[currentIndex - 1].title }}</a
        >
      </div>
```